streaming/base/format/mds/encodings.py

```diff
@@ -466,6 +466,28 @@ def decode(self, data: bytes) -> Image.Image:
 class JPEG(Encoding):
     """Store PIL image as JPEG."""
 
+    def __init__(self, quality: int = 75):
+        assert 0 <= quality <= 100
```

Comment:
nit: Please add an error message for this error itself

Reply:
please raise ValueError instead of assert. same for other places.